Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -79,7 +79,7 @@ std::pair<std::string, bool> GetNameArbPair(const TextureInfo& texture_info)
 
 void HiresTexture::Init()
 {
-  Update();
+  Clear();
```

Comment:
I was debating if this function should just be a stub, but figured better to be safe with Clear() when the VideoBackend init occurs. This happens before the title load change, so there is never a conflict.

I was unsure because of these members:
https://github.com/dolphin-emu/dolphin/pull/13244/files#diff-9c11da804dac15819708ea4654cc6acac008a862f5807f2097487e1c0e13add4R35-R38

If we want to just return/stub this function, that works too.

Reply:
Update: Opted to remove Init entirely